src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule73wherejavadocrequired/InputMissingJavadocTypeCheckNoViolations.java

```diff
@@ -0,0 +1,25 @@
+package com.google.checkstyle.test.chapter7javadoc.rule73wherejavadocrequired;
+
+/**
+ * This is a Javadoc comment.
+ */
+public class InputMissingJavadocTypeCheckNoViolations { // OK
+
+    /**
+     * This is a Javadoc comment.
+     */
+    public class Inner implements MyInterface{ // OK
+
+    }
+
+
+    public enum MyEnum { // OK, google style does not require public enums to have javadocs
+
+    }
+
+    public interface MyInterface {  // OK, google style does not require
+                                    // public interfaces to have javadocs
+
+    }
```

Comment:
From travis:
![image](https://user-images.githubusercontent.com/31252532/99266542-ddb20d80-27f0-11eb-8016-155a21098c43.png)

Why are these warnings here? I do not see these locally:
![image](https://user-images.githubusercontent.com/31252532/99267288-bdcf1980-27f1-11eb-9694-ad90523274d8.png)